src/main/java/uk/gov/hmcts/reform/civil/model/sdo/SmallClaimsFlightDelay.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.civil.model.sdo;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.civil.enums.sdo.OrderDetailsPagesSectionsToggle;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class SmallClaimsFlightDelay {
+
+    private String input1;
```

Comment:
Please put specific names for text boxes

Reply:
Azam, we have renamed to be more meaningful as you suggested.